OpenRA.Mods.Common/Traits/World/MusicPlaylist.cs

```diff
@@ -49,6 +52,8 @@ public class MusicPlaylist : INotifyActorDisposing, IGameOver, IWorldLoaded, INo
 
 		public readonly bool IsMusicInstalled;
 		public readonly bool IsMusicAvailable;
+		public readonly bool AllowMuteBackgroundMusic;
+		public Func<bool> IsBackgroundMusicMuted;
```

Comment:
Do we need this to not be a normal method or (maybe better) property?

Reply:
I don't know, I'm still learning... Whatever suits best. Do you mean a property with a public getter and private setter?

Reply:
I was thinking something like
```csharp
public bool IsBackgroundMusicMuted
{
	get { return AllowMuteBackgroundMusic && Game.Settings.Sound.MuteBackgroundMusic; }
}
```
and then removing the () after all the `IsBackgroundMusicMuted` calls.